### ISSUE TITLE
Delete all files in local directory

### ISSUE THREAD

**Issue by germancuacci** [bcedu/VGrive#96]
**Describe the bug**
Sometimes my Google Drive directory is empty and I have to log into the app again to download all the files again.
My internet connection is not working properly lately, i suspect that is the problem.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Fedora 30
 - VGrive Version 1.6 installed from repo fedora.

**Comment by j3frsn** [comment]
This just happened to me, but instead of only deleting my local files and syncing them again from Google Drive, the app is also deleting everything in the cloud. I will try logging out and back in again to see if is works.

**Comment by PaoloPelloni** [comment]
Indeed, it happens when it starts without an internet connection (or anyway it cannot reach the cloud). Lucky for me it never erased data in the cloud... but it's quite annoying anyway.
